samples/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ add_sample(TARGET placeholder-accessors SOURCES placeholder-accessors.cpp)
 add_sample(TARGET reduction SOURCES reduction.cpp)
 add_sample(TARGET reinterpret SOURCES reinterpret.cpp)
 add_sample(TARGET scan SOURCES scan.cpp)
+if (COMPUTECPP_SDK_USE_SUBGROUPS)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
I don't suppose it's really a problem, but this won't be scoped by the `if`, right?  So files before this won't be C++17 and ones after will be, it's a bit arbitrary.

Reply:
#270 has this fix

Reply:
Cool, thanks.